### PR TITLE
Index parent ID when indexing a collection

### PR DESCRIPTION
**Pull request by tpendragon**

### CODE REVIEW

File: app/models/spotlight/resources/iiif_manifest.rb [hunk 1]
```diff
@@ -36,6 +39,16 @@ def add_document_id
         solr_hash[exhibit.blacklight_config.document_model.unique_key.to_sym] = compound_id
       end
 
+      def add_parent_id
+        if parent
+          solr_hash[parent_id_field] = [parent.to_solr[exhibit.blacklight_config.document_model.unique_key.to_sym]]
```
Comment (cbeer) [inline app/models/spotlight/resources/iiif_manifest.rb]: Maybe we should expose `parent.to_solr[exhibit.blacklight_config.document_model.unique_key.to_sym]` through a nice helper, so we can skip unnecessarily creating the solr document?

**Comment by tpendragon** [comment]
@cbeer Look okay?

File: app/models/spotlight/resources/iiif_manifest.rb (hunk 1 above)
Comment (tpendragon) [inline app/models/spotlight/resources/iiif_manifest.rb]: Or I can just make compound_id public?

**Comment by tpendragon** [comment]
@cbeer Made #compound_id public so it doesn't have to generate that solr doc.

**Comment by cbeer** [comment]
👏

**Pull request merged**